### PR TITLE
add designate barclamp (SCRD-8739)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

**Comment by csymons-suse** [comment]
supersedes https://github.com/SUSE-Cloud/doc-cloud/pull/909/

File: xml/depl_nodes.xml [hunk 1]
```diff
@@ -150,6 +156,76 @@
...
+    create and propagate zones and records over the network using pools of DNS
+    servers. Deployment defaults are in place, so not much is required to
+    configure &desig;. &o_netw; needs additional settings for integration with
+    &desig;. The &desig; barclamp does not configure these &o_netw; settings; an
+    admin can enable them if necessary. For more information, see the <link
+    xlink:href="https://docs.openstack.org/designate/latest/contributor/integrations.html">OpenStack
+    Integrations</link>.
+  </para>
+  <para>
+    This designate barclamp relies heavily on dns barclamp and expects
```
Comment (dmpop) [inline xml/depl_nodes.xml]: designate -> &desig;

File: xml/depl_nodes.xml [hunk 2]
```diff
@@ -150,6 +156,76 @@
...
+          Installs the &desig; server packages and configures the mini-dns (mdns)
+          service required by &desig;.
+        </para>
+      </listitem>
+    </varlistentry>
+    <varlistentry>
+      <term>designate-worker role</term>
+      <listitem>
+       <para>
+        Configures a designate worker on the selected nodes. &desig; uses the
```
Comment (dmpop) [inline xml/depl_nodes.xml]: designate -> &desig;

Comment (csymons-suse) [reply xml/depl_nodes.xml]: both fixed

**Comment by csymons-suse** [comment]
held up for related pull request, see https://jira.suse.de/browse/SCRD-8739

File: xml/depl_nodes.xml [hunk 3]
```diff
@@ -150,6 +156,76 @@
...
+    create and propagate zones and records over the network using pools of DNS
+    servers. Deployment defaults are in place, so not much is required to
+    configure &desig;. &o_netw; needs additional settings for integration with
+    &desig;. The &desig; barclamp does not configure these &o_netw; settings; an
+    admin can enable them if necessary. For more information, see the <link
+    xlink:href="https://docs.openstack.org/designate/latest/contributor/integrations.html">OpenStack
+    Integrations</link>.
+  </para>
+  <para>
+    The &desig; barclamp relies heavily on dns barclamp and expects
```
Comment (asettle) [inline xml/depl_nodes.xml]: DNS

**Pull request merged**